Source/SPTableTextFieldCell.m

```diff
@@ -32,6 +32,46 @@
 
 @implementation SPTableTextFieldCell
 
+
+
+/**
+ * Initialise
+ */
+- (id) initWithCoder:(NSCoder *)coder
+{
+	self = [super initWithCoder:coder];
+	if (self) {
+//		noteButton = nil;
+		noteButton = [[NSCell alloc] init];
```

Comment:
Let's put noteButton into the lazy loaded property so you don't need this `initWithCoder` at all.